src/themes/website/styles/docs.scss

```diff
@@ -480,3 +484,43 @@ oauth-server-configuration {
         margin-top: 20px;
     }
 }
+
+.details-button {
```

Comment:
Looking at usage of `.details-button` and `.clickable`, it seems that it's artifacts from previous implementation. Do we still need these classes?

Also, `clickable` is always a link, we should allow user to define the color. With this class you override whatever colors a user sets.